index.js

```diff
@@ -1,20 +1,23 @@
-require('dotenv').config()
+require('dotenv').config();
 const express = require('express');
 const path = require('path');
 const cors = require('cors');
 const app = express();
-const Henesis = require('@haechi-labs/henesis-sdk-js').default;
+const { TransactionTracker } = require('@haechi-labs/henesis-sdk-js');
 const { Sender } = require('./helper/Sender');
 
 const transactions = {};
 
 const { CLIENT_ID, PRIVATE_KEY, NODE_ENDPOINT } = process.env;
-const henesis = new Henesis(CLIENT_ID);
+const tracker = new TransactionTracker(CLIENT_ID, {
```

Comment:
Plz add `PLATFORM`, `NETWORK` to `.env`.
And code is
```typescript
const { CLIENT_ID, PRIVATE_KEY, NODE_ENDPOINT, PLATFORM, NETWORK } = process.env;
const transactionTracker = new TransactionTracker(CLIENT_ID, {
    platform: PLATFORM,
    network: NETWORK
});
```

Reply:
I think that blockchain platform and network could be changed frequently. Therefore, it would be better to use hard coding rather than putting the variables as environment variables and using them statically.